### DISCUSSION TITLE
Menu size probbleem

### DISCUSSION THREAD

**Discussion by vnlrob** [notgoodusename/OsirisAndExtra#499] [Q&A] [answered]
Menu elements can be scrolled/get's a scroll bar when changing menu's size (and i dont want that ofc :D)

i use these 
ImGui::BeginChild("##Back", ImVec2{ 780, 545 }, false);
ImGui::BeginChild("##Main", ImVec2{ 785, 548 }, false);
ImGui::BeginChild("##UP", ImVec2{ 780, 548 }, false);
ImGui::BeginChild("##Childs", ImVec2{ 780, 548 }, false);
ImGui::BeginChild("##SubMain", ImVec2{ 780, 548 }, false);
i might be handicapped or smth cuz i cant get it to look right...

**Comment by Xsintashi** [comment, marked as answer]
use windowflags to disable scrolling and scrollbar
